samples/src/Lazy/LazyAspect.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AspectInjector.Broker;
+using System;
+using System.Collections.Generic;
+
+namespace Aspects.Lazy
+{
+    [Aspect(Scope.PerInstance)]

```

Comment:
this won't work for static properties. I mean there is only one instance of this aspect for all static properties in all classes.
So if there are two static property with the same name - there will be an issue
```c#
class A {
[Lazy]
static string Text => "aaaa"
}

class B {
[Lazy]
static string Text => "bbbb"
}
```
will both return aaaa or bbbb depending on which you call first

Reply:
I still believe we should make a key based on MethodBase (Source.Metadata) and instance if present. It will help to avoid such issues

Reply:
I just want better performance.
`Source.Metadata` is a reflection operation and exectued everytime.

Reply:
> I mean there is only one instance of this aspect for all static properties in all classes.

1. I know now, the static member use a Aspect singletone instance which is defined in Aspect class.

2. I previously thought that static properties did not require deferred loading.

    * static property initialized in .cctor is good:

        ```csharp
        public static string S { get; set; } = "";
        ```

    * static getonly property is just like static method, it doesn't need return one value everytime.

        ```csharp
        public static string SMethod => "";
        ```

Now, `LazyAspect` can used for static properties.



And I used "{Source.Type.FullName}.{Source.Name}" as key instead of `Source.Metadata`.